scripts/azure_cleanup.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import sys
+import traceback as tb
+from datetime import datetime
+
+from utils.path import log_path
+from utils.providers import list_provider_keys, get_mgmt
+
+
+def parse_cmd_line():
+    parser = argparse.ArgumentParser(argument_default=None)
+    parser.add_argument('--nic-template',
+                        help='NIC Name template to be removed', default="test*", type=str)
+    parser.add_argument('--pip-template',
+                        help='PIP Name template to be removed', default="test*", type=str)
+    parser.add_argument("--output", dest="output", help="target file name, default "
+                                                        "'cleanup_azure.log' in "
+                                                        "utils.path.log_path",
+                        default=log_path.join('cleanup_azure.log').strpath)
+    args = parser.parse_args()
+    return args
+
+
+def azure_cleanup(nic_template, pip_template, output):
+    with open(output, 'w') as report:
```

Comment:
Yep, we have the artifact in jenkins, and the file in the workspace can just be overwritten.